simple-rest-server/remove.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+)
+
+func remove(deleteID string) error {
+	buf, err := ioutil.ReadFile("fake.csv")
+	if err != nil {
+		fmt.Println(err)
+		return err
+	}
+	str := string(buf)
+	Lines := strings.Split(str, "\n")
+	var result []string
+	for i, v := range Lines {
```

Comment:
I've got a findIndexByID in checkout.go - I'll move it to Main.go and you should be able to use it here.